app/viewModels/drafts/AllDraftDeparturesViewModel.scala

```diff
@@ -32,7 +32,9 @@ case class AllDraftDeparturesViewModel(items: DeparturesSummary) {
   def visuallyHiddenHeader(implicit messages: Messages): String = messages(s"$messageKeyPrefix.heading.hidden")
 
   def referenceNumber(implicit messages: Messages): String = messages(s"$tableMessageKeyPrefix.lrn")
-  def daysToComplete(implicit messages: Messages): String  = messages(s"$tableMessageKeyPrefix.daysToComplete")
+
+  val lrnRedirectLocation                                 = s"$draftDepartureFrontendUrl/drafts/"
```

Comment:
Probably want the concatenation logic to be part of the viewmodel rather than the view itself so this might be better as `def lrnRedirectLocation(lrn: LocalReferenceNumber) = s"$draftDepartureFrontendUrl/drafts/$lrn"` then the line in the view file would become href="@lrnRedirectLocation(row.lrn)". Or could include the location as a field in the `DraftDepartureRow` case class.